dacbench/abstract_benchmark.py

```diff
@@ -201,9 +203,12 @@ def read_config_file(self, path):
         if "observation_space_type" in self.config:
             # Types have to be numpy dtype (for gym spaces)s
             if type(self.config["observation_space_type"]) == str:
-                typestring = self.config["observation_space_type"].split(" ")[1][:-2]
-                typestring = typestring.split(".")[1]
-                self.config["observation_space_type"] = getattr(np, typestring)
+                if self.config["observation_space_type"] == "None":
+                    self.config["observation_space_type"] = None
```

Comment:
Could you maybe tell me why you'd want to do that? Normally this is not desired behaviour, envs should always have an observation space. Is there maybe another way around this?

Reply:
Hi Theresa,

This is the observation space type, not the space itself I think. In fact it was specified as None in the current [CMAES](https://github.com/automl/DACBench/blob/c74da0308a19d4828d171bdfde4773c01b850c93/dacbench/benchmarks/cma_benchmark.py#L26
) default configuration because the space is a dictionary and doesn't have a unique type for all elements.

The above change is to avoid error when we save a config to .json file and then load it again from file, as in my example code in Issue 1.

Reply:
Oh, sorry, that's a bug and I misread what you did there. Thanks for noticing and fixing, the logging system still needs some work as you can see. 
I'll merge both requests in that case :D

Reply:
No problemo! :D Thanks for reviewing and merging it so quickly!

Nguyen